### PR TITLE
Scripts that pull sign and pushes the nfs-ganesha packages on the signer

### PR DESCRIPTION
**Pull request by akraitman**
Signed-off-by: Adam Kraitman <akraitma@redhat.com>

### CODE REVIEW

File: scripts/nfs-ganesha/pull-sign-push [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# After this the sign-rpms-auto script will run and sign the nfs-ganesha packages
+# And finally the sync-push-auto script will run and will push the signed packages to download.ceph.com
+
+today_items=$(ssh ubuntu@chacra.ceph.com 'find /opt/repos/nfs-ganesha-stable -newermt "-24 hours" -ls' | awk '{ print $11 }' )
+if [ -n "$today_items" ]; then
+echo "pulling nfs-ganesha packages from chacra"
+echo "********************************************"
+[[ -d /opt/new-repos/ ]] | mkdir -p /opt/new-repos/
+  for item in $today_items; do
+   sync_cmd="ubuntu@chacra.ceph.com:$item /opt/new-repos/"
```
Comment (djgalloway) [inline scripts/nfs-ganesha/pull-sign-push:12]: Can you make this something like `/opt/nfs-ganesha/new-repos` so it's clear what sort of repos they are?

Comment (akraitman) [reply scripts/nfs-ganesha/pull-sign-push]: Yes changed the folder name

File: scripts/nfs-ganesha/pull-sign-push [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+[[ -d /opt/nfs-ganesha/new-repos/ ]] | mkdir -p /opt/nfs-ganesha/new-repos/
+  for item in $today_items; do
+   sync_cmd="ubuntu@chacra.ceph.com:$item /opt/nfs-ganesha/new-repos/"
+   rsync -Lavh --progress --relative $sync_cmd
+  done
+
+  # sign the rpm's that wore pulled today
+
+echo "signing rpms"
+bash ./sign-rpms-auto
```
Comment (djgalloway) [inline scripts/nfs-ganesha/pull-sign-push:19]: Can you make this an absolute path, please?

File: scripts/nfs-ganesha/pull-sign-push [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  # syncing the singed rpm's to download.ceph.com
+
+echo "pushing rpms to download.ceph.com"
+bash ./sync-push-auto
```
Comment (djgalloway) [inline scripts/nfs-ganesha/pull-sign-push:24]: Can you make this an absolute path, please?

**Review by djgalloway — APPROVED** [review]

**Pull request merged**